erp/views.py

```diff
@@ -256,6 +259,26 @@ def _get_or_create_api_key():
     return api_key
 
 
+def get_equipments_groups():
+    # TODO add icon
+    EquipmentGroup = namedtuple("EquipmentGroup", ["name", "slug", "equipements"])
+
+    # TODO example values, replace by correct values
+    for_pmr = EquipmentGroup(
```

Comment:
This is purely for demonstration, I did not follow any logic or what is planned in the miro document